contrib/linux/actions/service.py

```diff
@@ -70,6 +70,7 @@ def get_linux_distribution():
     re.search(distro, "Redhat")
     or re.search(distro, "Fedora")
     or re.search(distro, "CentOS Linux")
```

Comment:
```suggestion
    or re.search(distro, "CentOS")
```
I guess that's just a diff we could include here

For the core `linux` pack changes, I think we'll need to bump the pack version as well?